requirements/PROCESSES.md

```diff
@@ -0,0 +1,89 @@
+The document for the first phase should be structured as follows:
+
+# Introduction
+Brief overview.
+
+# Goals
+High-level descrition of project goals.
+
+# Usage
```

Comment:
Hm, I just started trying to split up and write a pair of sentences here. I think that Usage Environment may be just a single Functional requirement (there's not much to say here after all). And Usage scenarios actually belong to their own section I'd say.